packages/portal-proto/src/components/Modals/UserProfileModal.tsx

```diff
@@ -72,7 +72,7 @@ export const UserProfileModal = ({
       }
       openModal={openModal}
       size="60%"
-      buttons={[{ title: "Done", dataTestId: "button-user-profile-done" }]}
+      buttons={[{ title: "Close", dataTestId: "button-user-profile-done" }]}
```

Comment:
I feel like this might not be correct because the modal shows two different things. Can we say `Done` for when the data length is greater than 0 and `Close` for the other case?

cc. @wteouchicago